player_darwin.c

```diff
@@ -0,0 +1,143 @@
+//
+// Created by Christopher Cooper on 7/12/17.
+//
+
+#include <sys/param.h>
+#include "player_darwin.h"
+
+#pragma mark - utility functions
```

Comment:
Pragmas are very common on OSX like this. I can take them out if you don't like them.